h2o-py/h2o/model/metrics_base.py

```diff
@@ -567,6 +569,10 @@ def null_degrees_of_freedom(self):
             return self._metric_json["null_degrees_of_freedom"]
         return None
 
+    # private accessor for mean per-class error - the public version is overridden in H2OBinomialModelMetrics with
+    # a method with different return semantics
```

Comment:
so it means that for binomial tasks, we always print the backend value only, right?
This same name with different semantics is confusing indeed.

Btw, `H2OBinomialModelMetrics` doesn't exist, it's `H2OBinomialModel` mixin in Py and `ModelMetricsBinomial` on backend.

Reply:
> so it means that for binomial tasks, we always print the backend value only, right?
yes, we print the backend values without re-calculating the same thing

> Btw, H2OBinomialModelMetrics doesn't exist, it's H2OBinomialModel mixin in Py and ModelMetricsBinomial on backend.

I think it does - https://github.com/h2oai/h2o-3/blob/master/h2o-py/h2o/model/metrics_base.py#L850:

```
class H2OBinomialModelMetrics(MetricsBase):
    """
    This class is essentially an API for the AUC object.
    This class contains methods for inspecting the AUC for different criteria.
    To input the different criteria, use the static variable `criteria`.
    """
```

Reply:
my bad! this `metrics_base` module has many many classes :)